modules/nf-core/msisensor2/msi/main.nf

```diff
@@ -42,7 +42,7 @@ process MSISENSOR2_MSI {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        msisensor2: \$(echo \$(msisensor2 2> >(grep Version) | sed 's/Version: v//g'))
+        msisensor2: \$(echo \$(msisensor2 2>| >(grep Version) | sed 's/Version: v//g'))
```

Comment:
I don't think it matters in this one since it's in a subshell. The author used an unusual way of getting the version here.